packages/gatsby/src/utils/api-node-docs.ts

```diff
@@ -133,6 +133,17 @@ export const sourceNodes = true
  */
 export const onCreateNode = true
 
+/**
+ * Called when a new node is created before the `onCreateNode` handler is called.
+ * This is an optimization that can prevent the `onCreateNode` handler to be scheduled
+ * if the plugin already knows it's not interested in processing this node.
+ *
+ * @gatsbyVersion 2.24.78
```

Comment:
This probably is going to be 79, as we are currently on 78
```suggestion
 * @gatsbyVersion 2.24.79
```

Reply:
Actually I just realized that this warning works the other way around.

If your Gatsby version is new enough then this warning will never be emitted. If your plugin is older than when the feature was introduced then the warning won't be emitted. If your plugin is newer but your Gatsby is not new enough then the generic warning is shown, because it won't ship with this particular information either. So any way I look at it, this version is essentially dead code.

I'm fine keeping it, but I think it's worth pointing out that this warning system will not print a nice warning for a certain set of new features.

Reply:
> If your plugin is newer but your Gatsby is not new enough then the generic warning is shown, because it won't ship with this particular information either. So any way I look at it, this version is essentially dead code.

No, the old core versions will try to pull produced API file from `unpkg` to give most up to date information possible (even if gatsby itself is much older).

Reply:
https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/src/utils/get-latest-apis.ts#L15-L35

Reply:
Oh, that's awesome then. Nice!